Mantle/MTLJSONAdapter.m

```diff
@@ -370,13 +370,9 @@ + (NSDictionary *)valueTransformersForModelClass:(Class)modelClass {
 	for (NSString *key in [modelClass propertyKeys]) {
 		SEL selector = MTLSelectorWithKeyPattern(key, "JSONTransformer");
 		if ([modelClass respondsToSelector:selector]) {
-			NSInvocation *invocation = [NSInvocation invocationWithMethodSignature:[modelClass methodSignatureForSelector:selector]];
-			invocation.target = modelClass;
-			invocation.selector = selector;
-			[invocation invoke];
-
-			__unsafe_unretained id transformer = nil;
-			[invocation getReturnValue:&transformer];
+			IMP imp = [modelClass methodForSelector:selector];
+			NSValueTransformer*(*function)(id, SEL) = (void *)imp;
```

Comment:
Can you please space out the pointer asterisk like so:

```objc
NSValueTransformer * (*function)(id, SEL)
```

to more closely match how one would write a variable declaration? :pray: